httpx/_client.py

```diff
@@ -1010,10 +1010,13 @@ def _send_single_request(self, request: Request) -> Response:
         self.cookies.extract_cookies(response)
         response.default_encoding = self._default_encoding
 
-        status = f"{response.status_code} {response.reason_phrase}"
-        response_line = f"{response.http_version} {status}"
-        logger.debug(
-            'HTTP Request: %s %s "%s"', request.method, request.url, response_line
+        logger.info(
```

Comment:
Based on the other python packages, I don't recall any other python client having an INFO log message level. :thinking:

Reply:
i.e. can't it be debug as well?

Reply:
It could yes, though it's less useful to have logging if everything is at the same level.
I like the different "show me nothing", "show me the requests", "show me the debug" lighting levels.

(Similar: gunicorn, uvicorn using INFO for requests, and DEBUG for other stuffs)